script/sync_prod_to_gcp/submissions_to_gcp.py

```diff
@@ -532,6 +532,12 @@ def is_html_submission(self) -> bool:
         assert(self.source_format)
         return self.source_format == "html"
 
+    @property
+    def is_ps_submission(self) -> bool:
+        """is a HTML submission"""
```

Comment:
That comment should be "is a PostScript submission" or similar

Reply:
oops.